### ISSUE TITLE
Keepassxc 2.7.1 auto-type logs me out of the current user’s session

### ISSUE THREAD

**Issue by rukyasu** [keepassxreboot/keepassxc#8047]
## Overview
If I try to perform auto-type, I get immediately logged out of the current user’s session. Previously used 2.3.4 didn't cause this for me.

## Steps to Reproduce
1. Start auto-type via global shortcut or manually via entry context menu

## Expected Behavior
It performs auto-type.

## Actual Behavior
It log's me out of the current user’s session.

## Context
KeePassXC - Version 2.7.1
Revision: 5916a8f
Distribution: AppImage

Qt 5.15.2
Debugging mode is disabled.

Operating system: Debian GNU/Linux 10 (buster)
CPU architecture: x86_64
Kernel: linux 5.4.0-0.bpo.4-amd64

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- Botan 2.12.1

**Comment by droidmonkey** [comment]
You must have chosen an Auto-Type shortcut that is actually the shortcut to log off your machine. Choose another one.

**Comment by rukyasu** [comment]
Thank you for the reply. If you mean "Global Auto-Type shortcut", I set an unoccupied shortcut. It logs me out regardless of the method. Right click on an entry > Perform Auto-Type > {USERNAME}{TAB}{PASSWORD}{ENTER} does the same thing. I wanted to check logs but in another thread I've found keepassxc doesn't log much.

EDIT: A detail - any of the options in Perform Auto-Type context menu behaves this way (only {USERNAME}, {USERNAME}{ENTER}, etc...)
